rb/lib/selenium/webdriver/common/virtual_authenticator_options.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+#
+# Options for the creation of virtual authenticators.
+# @see http://w3c.github.io/webauthn/#sctn-automation
+#
+#
+module Selenium
+  module WebDriver
+    class VirtualAuthenticatorOptions
+
+      PROTOCOL = {"CTAP2" => "ctap2", "U2F" => "ctap1/u2f"}.freeze
```

Comment:
Usually Ruby would have keys as symbols, so let's use those instead:
```suggestion
      PROTOCOL = {ctap2: "ctap2", u2f: "ctap1/u2f"}.freeze
```

Reply:
Done. Thank you for the heads up.